OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -119,6 +151,22 @@
     </codeblock>
 </div>
 
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="CTL-x100 Android Mode">
+    Wacom CTL-x100 Android Mode
+</h3>
+<div class="ms-3">
+<p>
+    Sometimes the Ctl-x100s can be left in Android mode instead of PC mode, to fix this press
```

Comment:
```suggestion
    Sometimes Ctl-x100 tablets can be started in Android mode instead of PC mode. To fix this, press
```

Reply:
Continuing from Lime's change (capitalize model):
```suggestion
    Sometimes CTL-x100 tablets can be started in Android mode instead of PC mode. To fix this, press
```

Reply:
section replaced by voids better wording.